### PR TITLE
changes for UI build and deployment

### PR DESCRIPTION
**Pull request by naveen-egov**
Build Changes 

- Added Path strategy /mgramseva/
- build image Name - frontend-mgramseva

### CODE REVIEW

File: build/build-config.yml [hunk 1]
```diff
@@ -48,8 +48,8 @@ config:
       - work-dir: "echallan-calculator"
         image-name: "echallan-calculator"
         dockerfile: "build/maven/Dockerfile"
-  - name: "builds/mGramSeva/frontend/egov-web-ui"
+  - name: "builds/mGramSeva/frontend/frontend-mgramseva"
     build:
       - work-dir: "frontend/mgramseva"
-        image-name: "egov-web-ui"
+        image-name: "frontend-mgramseva"
         dockerfile: "frontend/mgramseva/Dockerfile"
```
Comment (nikesh-eGov) [inline build/build-config.yml:55]: Set correct dockerfile path
frontend/mgramseva/docker/Dockerfile

Comment (naveen-egov) [reply build/build-config.yml]: Updated the Code.

**Review by nikesh-eGov — CHANGES_REQUESTED** [review]
Everywhere replace the frontend-mgramseva with the mgramseva.

File: frontend/mgramseva/docker/Dockerfile [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
+RUN flutter build web
 # Create runtime image
 FROM nginx:mainline-alpine
-ENV WEB_DIR=/var/web/
-
-RUN mkdir -p ${WEB_DIR}
-
-COPY --from=build /app/packages/employee/build ${WEB_DIR}/
\ No newline at end of file
+ENV WEB_DIR=/var/web/frontend-mgramseva
```
Comment (nikesh-eGov) [inline frontend/mgramseva/docker/Dockerfile]: change frontend-mgramseva with mgramseva

File: frontend/mgramseva/docker/nginx.conf [hunk 3]
```diff
@@ -0,0 +1,12 @@
+server
+{
+  listen 80;
+  underscores_in_headers on;
+  server_tokens off;
+  location /mgramseva
+  {
+    root /var/web;
+    index index.html index.htm;
+    try_files $uri $uri/ /frontend-mgramseva/index.html;
```
Comment (nikesh-eGov) [inline frontend/mgramseva/docker/nginx.conf]: here as well change frontend-mgramseva with mgramseva

**Review by nikesh-eGov — APPROVED** [review]

**Pull request merged**